docs/adoc/microfrontend-platform-developer-guide/chapters/configuration/platform-configuration/platform-configuration.adoc

```diff
@@ -120,6 +131,18 @@ a| `Array<ApplicationConfig>`
 | yes
 | Defines the micro applications running in the platform. See <<chapter:platform-configuration:application-config,ApplicationConfig>> for more information.
 
+| manifestLoadTimeout
+a| `number`
+| no
+| Optional global timeout for loading the manifest of each application in milliseconds. +
+By default, the timeout is defined by the browser. Overriding this value can speed up the platform startup process, if unavailable micro apps take long to respond (e.g. web application firewall is delaying the response).
+
+| activatorLoadTimeout
+a| `number`
+| no
+| Optional global timeout for each activator readiness check in milliseconds. +
+By default, the platform will wait indefinitely. This will block the platform startup process if an activator never signals ready.
```

Comment:
...will wait indefinitely. This will block the platform...
-->
Sollte das ev. in present tense sein? Ist es klar, dass sich "This will block" auf den vorhergehenden Satz und nicht auf das Property bezieht?
Ev. so besser: waits indefinitely, which blocks the platform